src/node.h

```diff
@@ -870,7 +898,8 @@ class NODE_EXTERN CommonEnvironmentSetup {
       MultiIsolatePlatform* platform,
       std::vector<std::string>* errors,
       const std::vector<std::string>& args = {},
-      const std::vector<std::string>& exec_args = {});
+      const std::vector<std::string>& exec_args = {},
+      const SnapshotConfig& snapshot_config = {});
```

Comment:
Technically ABI-breaking, right? If you keep it this way, I’m happy to do an ABI-compatible backport to 20.x

Reply:
Yeah I think we can break it here and duplicate the original signature in v20.x.